drbert/run.py

```diff
@@ -57,8 +68,6 @@ def prepare_optimizer_and_scheduler(args, model, t_total):
     decay_blacklist = {'LayerNorm.bias', 'LayerNorm.weight'}
 
     decay, no_decay = [], []
-
-    for name, param in model.named_parameters():
```

Comment:
@JohnGiorgi I think this line got deleted by accident

Reply:
Yes, nice catch!